Firestore/core/test/unit/model/object_value_test.cc

```diff
@@ -85,60 +72,60 @@ TEST_F(ObjectValueTest, ExtractsFieldMask) {
 }
 
 TEST_F(ObjectValueTest, OverwritesExistingFields) {
-  MutableObjectValue object_value = WrapObject("a", "object_value");
+  ObjectValue object_value = WrapObject("a", "object_value");
   EXPECT_EQ(WrapObject("a", "object_value"), object_value);
-  object_value.Set(Field("a"), Wrap("object_value"));
+  object_value.Set(Field("a"), Value("object_value"));
```

Comment:
Note: Value is the function name in `master`, so this actually reduces the diff.